test/OpenTelemetry.Instrumentation.Grpc.Tests/Services/GreeterService.cs

```diff
@@ -31,7 +31,7 @@ public GreeterService(ILoggerFactory loggerFactory)
 
         public override Task<HelloReply> SayHello(HelloRequest request, ServerCallContext context)
         {
-            this.logger.LogInformation($"Sending hello to {request.Name}");
+            this.logger.LogInformation("Sending hello to {Name}", request.Name);
```

Comment:
Nice! 👍

Reply:
🍨 for anyone who migrates to do structured logging only :)